scripts/commands/softgodmode.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- func: SoftGodMode
+-- desc: Toggles a toned-down version of godmode that allows the GM to test incoming/outgoing damage
```

Comment:
We've got `!immortal` for this

Reply:
Immortal just makes you impossible to die. I found this command to be extremely useful for my testing on Wings if I wanted to test outgoing damage without all the huge damage boosts that godmode gives you.

Reply:
What is it you need from this exactly? Immortal leaves your damage normal, and you can set mp and tp as needed if you need to test ws or spells. Or just `!addeffect regain 1000 0 0`  instead of setting tp. Just trying to understand the reason.

Reply:
I guess I just found it to make testing easier without having to spam commands so much or add effects one by one. It was just a one command that would give me everything I needed to tune and balance fights/NMs.

Reply:
Additional thought: we could add an optional parameter to the existing godmode to set "tiers" of it. It looks like this version you have is lost on zone intentionally so it wouldn't clutter up the player.lua so I don't think anyone would oppose that modification

Reply:
+1 for the tiers. I think it's best to remove this from this PR and we'll talk about reworking `godmode` to be more flexible later on

Reply:
Please take a look, I've gone ahead and added some adjustments to godmode to take the argument of `1` to enable this version of soft god mode.